portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,71 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+/*NAVBAR*/
+
+.navbar {
+  overflow: hidden;
+  background-color: #333;
+  top: 10;
+  /*potion: fixed;*/
```

Comment:
delete the commented out property if it is not used. It is best not to submit unused or temprorary code. This keeps the main repo clean

Reply:
done